src/App.js

```diff
@@ -1,19 +1,45 @@
 import React from 'react';
 import './App.css';
-
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
+import { Container } from 'react-bootstrap';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoForm } from './components/TodoForm/TodoForm';
 import users from './api/users';
+import todos from './api/todos';
+
+const prepearedtodos = todos.map(todo => ({
+  ...todo,
+  userName: users.find(user => user.id === todo.userId).name,
+}));
+
+class App extends React.Component {
+  state = {
+    todosOnPage: prepearedtodos,
```

Comment:
```suggestion
    todos: preparedTodos,
```

Reply:
Unfortunately, the linter shows an error:
```
 error  'todos' is already declared in the upper scope 
 ```
 So I'll leave this variable name